website/docs/r/alert_channel_splunk.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "Alert Channels"
+layout: "lacework"
+page_title: "Lacework: lacework_alert_channel_splunk"
+description: |-
+  Create and manage Splunk Alert Channel integrations
+---
+
+# lacework\_alert\_channel\_splunk
+
+You can use this resource enabled Lacework to forward alerts to Splunk using an HTTP Event Collector.
+
+To find more information see the [Lacework support documentation](https://support.lacework.com/hc/en-us/articles/360007889274-Splunk).
+
+## Example Usage
+
+```hcl
+resource "lacework_alert_channel_" "ops_critical" {
+  name      = "OPS Critical Alerts"
+  hec_token = "AA111111-11AA-1AA1-11AA-11111AA1111A"
+  host = "localhost"
+  port = "80"
+  event_data {
+    index = "index"
+    source = "source"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The Alert Channel integration name.
+* `channel` - (Optional) The Splunk channel name
+* `hec_token` - (Required) The token you generate when you create a new HEC input.
+* `host` - (Required) The hostname of the client from which you're sending data.
+* `port` - (Required) The destination port for forwarding events [80 or 443].
```

Comment:
I am not 100% sure that the port could/should be only 80 or 443. That sounds like what SSL option would do 🤔-- I will verify this value internally, I have a feeling that the documentations on our site are wrong.

Reply:
I'll remove for now.